app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/observables/FahrplanEmptyParameter.kt

```diff
@@ -0,0 +1,14 @@
+package nerd.tuxmobil.fahrplan.congress.schedule.observables
+
+import nerd.tuxmobil.fahrplan.congress.schedule.FahrplanFragment
+import nerd.tuxmobil.fahrplan.congress.schedule.FahrplanViewModel
+
+/**
+ * Payload of the observable [fahrplanEmptyParameter][FahrplanViewModel.fahrplanEmptyParameter]
+ * property in the [FahrplanViewModel] which is observed by the [FahrplanFragment].
+ */
+data class FahrplanEmptyParameter(
```

Comment:
:x: I could come up with a better name at that point. However, the class should clear up the general error UI mechanism (currently `AlertDialog` and `DialogFragment`) is cleaned up. Then it should be possible for a view model to emit a pure error message to be rendered. In sum, I would not work on this now.

Reply:
I cannot quite follow here. Do you mean this Parameter-pattern will go away or you mean only this particular object will change/will be removed?

Reply:
The view model will expose what is now returned by `ErrorMessage.Factory#getMessageForHttpStatus`.

Reply:
That still not entirely answers my question - do you plan to get rid off the Parameter -> SingleEvent -> View, or you mean it will change for this particular case. However I believe a common Interface which marks this as side effects would have help me to understand were they belong to, right away without follow back the trails.
But if you say, we postpone that/move that in a separate issue I am happy with that.

Reply:
The next steps for the data flow from view models to fragments/activities will be:

- Introduce a `Result<Payload>` type which wraps success, error, loading states. Currently separated `LiveData` properties will be combined.
- Later, `LiveData` will be replaced with `Flow`.

> Also the name "Parameter" is quite generic.

I wouldn't change the name **anymore now** that we are working on this screen. If somebody comes up with a better name all screens (view models), speaking of favorites, schedule changes, details, schedule should be adapted **in one step** to have a **consistent** wording across the project.

> ... a common Interface which marks this as side effects ...

I wouldn't say the properties are side effects. I see them as the primary reactive API of the view model plus the public functions. - Introducing a common interface might help with the discoverability. Here too, it would be good to have it consistent in all screens (view models). In case we want to have it - do you have a suggestion for a name. Please mind that some classes belong to the `MainViewModel` while others belong to the `FahrplanViewModel`.

Reply:
Maybe we can borrow from [Compose](https://developer.android.com/jetpack/compose/side-effects) here -> and call them `Effects` or `OneOffEvents`. It's generic enough to be independent of domain but telling which context it is used. 

In terms of constancy I surely agree...this is would be another  follow-up.

Reply:
Definitely, the wording should be well thought of. I pointed out in [this conversation](https://github.com/EventFahrplan/EventFahrplan/pull/424#discussion_r769906279) that common sense can be expected. 

I prefer to postpone introducing an interface as a follow-up because then it can be done to all view models at once. They might even share an interface.

Reply:
Deal!